### DISCUSSION TITLE
How to set tooltip for system tray icon?

### DISCUSSION THREAD

**Discussion by wkl007** [tauri-apps/tauri#4871] [Q&A] [answered]
like electron setToolTip: https://www.electronjs.org/docs/latest/api/tray#traysettooltiptooltip

![image](https://user-images.githubusercontent.com/23228335/182992578-77cd5fcc-15b0-4079-8d4a-9d4ab06a67d5.png)

**Comment by FabianLars** [comment, marked as answer]
Initial upstream support was just merged 3 days ago: https://github.com/tauri-apps/tao/pull/410. But it needs some time until we can expose it in Tauri.
Also note that it currently is macOS only, but Windows will be added before it lands in Tauri.
Libappindicator (Linux) doesn't seem to support it at all tho.

**Reply by wkl007** [reply]
Thank you very much

**Reply by ninetynin** [reply]
> Initial upstream support was just merged 3 days ago: [tauri-apps/tao#410](https://github.com/tauri-apps/tao/pull/410). But it needs some time until we can expose it in Tauri. Also note that it currently is macOS only, but Windows will be added before it lands in Tauri. Libappindicator (Linux) doesn't seem to support it at all tho.

Can I know if there is support for system tool tip in windows as i cant find anything in docs
